src/nanoemoji/write_font.py

```diff
@@ -257,16 +259,22 @@ def _create_glyph(color_glyph: ColorGlyph, painted_layer: PaintedLayer) -> Glyph
     return glyph
 
 
-def _draw_glyph_extents(ufo: ufoLib2.Font, glyph: Glyph):
+def _draw_glyph_extents(
+    ufo: ufoLib2.Font, glyph: Glyph, bounds: Optional[Tuple[float, float, float, float]]
```

Comment:
bounds seems like it should be a non-optional Rect?

Reply:
made it non-optional